src/editor/components/LessonSection.js

```diff
@@ -1,31 +1,114 @@
-import React from 'react';
-import LessonPlanTextInput from './LessonPlanTextInput';
-import { Text, View, SafeAreaView, StyleSheet } from 'react-native';
+import React, { useState } from 'react';
+import AddLessonContentButton from './AddLessonContentButton';
+import { Text, View, SafeAreaView, Platform, StyleSheet } from 'react-native';
+import store from '../../services/configureStore';
+import ContentCard from './ContentCard';
+import StoredContent from './StoredContent';
+
+const LessonSection = ({ sectionType, subtitle }) => {
+  const [sectionContent, setSectionContent] = useState([]);
+  const [sectionActivityCards, setSectionActivityCards] = useState([]);
+  const [isTextinputOpen, setisTextinputOpen] = useState(false);
+  const handleClick = () => {
+    setisTextinputOpen(true);
+  };
 
-const LessonSection = ({ subtitle }) => {
   return (
-    <SafeAreaView style={styles.sectionContainer}>
+    <SafeAreaView>
       <Text style={styles.title}>{subtitle}</Text>
-      <View>
-        <LessonPlanTextInput placeholder={'Input text'} />
-        <LessonPlanTextInput placeholder={'Add activity cards'} />
+      <View style={styles.sectionContainer}>
+        {/* New textbox with prompted to insert text */}
+        {isTextinputOpen && (
+          <ContentCard
+            setisTextinputOpen={setisTextinputOpen}
+            setSectionContent={setSectionContent}
+            sectionContent={sectionContent}
+            sectionType={sectionType}
+          />
+        )}
+
+        <AddLessonContentButton
+          placeholder={'Input text'}
+          handleClick={handleClick}
+          style={{ fontFamily: 'Poppins-Light' }}
```

Comment:
Is there a reason why this button specifically only has style? Also, it's not being passed in as a prop [here](https://github.com/uoftblueprint/sistema/blob/08b399d0ccbcd4af16d771a8848027cabfaf2518/src/editor/components/AddLessonContentButton.js#L11) (so I would either standardize it for all instances of this component or delete this line.